lib/interface_quda.cpp

```diff
@@ -5350,29 +5351,33 @@ void performAPEnStep(unsigned int n_steps, double alpha, int meas_interval)
   GaugeFieldParam gParam(*gaugeSmeared);
   auto *cudaGaugeTemp = new cudaGaugeField(gParam);
 
-  QudaGaugeObservableParam param = newQudaGaugeObservableParam();
-  param.compute_qcharge = QUDA_BOOLEAN_TRUE;
-
+  // The nth measurement to take
+  int measurement_n = 0;
+  
   if (getVerbosity() >= QUDA_SUMMARIZE) {
-    gaugeObservablesQuda(&param);
-    printfQuda("Q charge at step %03d = %+.16e\n", 0, param.qcharge);
+    gaugeObservablesQuda(&param[measurement_n]);
+    printfQuda("Q charge at step %03d = %+.16e\n", 0, param[measurement_n].qcharge);
   }
```

Comment:
Thank you for the changes. Is there a reason why the call to `gaugeObservablesQuda` is guarded by `if (getVerbosity() >= QUDA_SUMMARIZE)` here?

Reply:
Good catch, there was another misplaced `gaugeObservablesQuda` too.

Are you happy with this version works with TMLQCD?

Reply:
yes, thank you